### PR TITLE
analytics layer for daily finances

### PR DESCRIPTION
**Pull request by hyunjinjun**

### CODE REVIEW

File: models/analytics/daily_finances.sql [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    SELECT 
+        DATE(returned_at) as returned_date,
+        SUM(CASE WHEN is_refunded = 'yes' THEN total_price ELSE 0 END) as daily_refund,
+    FROM {{ ref('base_googledrive__returns') }}  r
+    JOIN {{ ref('int_orders_withprice') }}  o 
+    ON r.order_id = o.order_id
+    GROUP BY 1
+    )
+
+, expense as (
```
Comment (BobbyLiu1016) [inline models/analytics/daily_finances.sql:20]: 100% sure that the date variable is consecutive without any missing values

File: models/analytics/daily_finances.sql [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    )
+
+SELECT
+    o.order_date as _date,
+    MAX(o.daily_order_amt) as daily_order_amt,
+    MAX(o.daily_shipping_cost) as daily_shipping_cost,
+    MAX(r.daily_refund) as daily_refund,
+    MAX(e.daily_expense) as daily_expense,
+    MAX(h.daily_avg_salary) as daily_avg_salary,
+    MAX(o.daily_order_amt) - MAX(r.daily_refund) - MAX(e.daily_expense) - MAX(h.daily_avg_salary) as daily_profit
```
Comment (BobbyLiu1016) [inline models/analytics/daily_finances.sql]: COALESCE(X, 0) will be better?

File: models/analytics/daily_finances.sql [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+FROM orders o
+FULL OUTER JOIN refund r ON o.order_date = r.returned_date
+FULL OUTER JOIN expense e ON o.order_date = e.date
+, hr_cost h
```
Comment (BobbyLiu1016) [inline models/analytics/daily_finances.sql:44]: daily_profit should  minus this hr_cost?

Comment (BobbyLiu1016) [reply models/analytics/daily_finances.sql:44]: FROM expense
LEFT JOIN refund
LEFT JOIN orders

Comment (hyunjinjun) [reply models/analytics/daily_finances.sql:44]: hr_cost has been already deducted as 'daily_avg_salary'

File: models/analytics/daily_finances.sql (hunk 2 above)
Comment (hyunjinjun) [reply models/analytics/daily_finances.sql]: Added!

File: models/analytics/daily_finances.sql (hunk 1 above)
Comment (hyunjinjun) [reply models/analytics/daily_finances.sql:20]: switched the reference table from orders -> expense table

**Pull request merged**